js/components/graph/Graph.js

```diff
@@ -41,6 +39,12 @@ export default class Graph extends React.Component {
       infoBoxXPos: 0,
       infoBoxYPos: 0,
       infoBoxNodeId: "",
+      panning: false,
+      panStartX: 0,
+      panStartY: 0,
+      viewBoxPos: {x: 0, y: 0},
```

Comment:
Let's use one combined dictionary for all 4 viewbox states (x, y, width, height).